doc/rtd/development/style_docs.rst

```diff
@@ -12,7 +12,6 @@ be changed to UK English.
 Try to be concise and to the point in your writing. It is acceptable to link
 to official documentation elsewhere rather than repeating content. It's also
 good practice not to assume that your reader has the same level of knowledge
-as you, so if you're covering a new or complicated topic, then providing
```

Comment:
Was this line deletion intended?

Reply:
Not even remotely - no idea how I did that. Fixed!